runtime/logging_test.go

```diff
@@ -45,3 +50,167 @@ func TestDropInputParam(t *testing.T) {
 	}
 
 }
+
+func TestPrettyFormatterNoFields(t *testing.T) {
+	fmtr := prettyFormatter{}
+
+	e := logrus.NewEntry(logrus.StandardLogger())
+	e.Message = "test"
+	e.Level = logrus.InfoLevel
+
+	out, err := fmtr.Format(e)
+	if err != nil {
+		t.Fatalf("Unexpected error formatting log entry: %s", err.Error())
+	}
+
+	actualStr := string(out)
+
+	expectedLvl := strings.ToUpper(e.Level.String())
+	if !strings.Contains(actualStr, expectedLvl) {
+		t.Errorf("Expected log message to have level %s:\n%s", expectedLvl, actualStr)
+	}
+
+	if !strings.Contains(actualStr, "test") {
+		t.Errorf("Expected log message to have the entry message '%s':\n%s", "test", actualStr)
+	}
+}
+
+func TestPrettyFormatterBasicFields(t *testing.T) {
+	fmtr := prettyFormatter{}
+
+	e := logrus.WithFields(logrus.Fields{
+		"number": 5,
+		"string": "field_string",
+		"nil":    nil,
+		"error":  errors.New("field_error").Error(),
+	})
+
+	e.Message = "test"
+	e.Level = logrus.InfoLevel
+
+	out, err := fmtr.Format(e)
+	if err != nil {
+		t.Fatalf("Unexpected error formatting log entry: %s", err.Error())
+	}
+
+	actualStr := string(out)
+
+	expectedLvl := strings.ToUpper(e.Level.String())
+	if !strings.Contains(actualStr, expectedLvl) {
+		t.Errorf("Expected log message to have level %s:\n%s", expectedLvl, actualStr)
+	}
+
+	if !strings.Contains(actualStr, "test") {
+		t.Errorf("Expected log message to have the entry message '%s':\n%s", "test", actualStr)
+	}
+
+	if !strings.Contains(actualStr, "number = 5") {
+		t.Errorf("Expected to have the number field in message")
+	}
+
+	if !strings.Contains(actualStr, "string = \"field_string\"") {
+		t.Errorf("Expected to have the string field in message")
+	}
+
+	if !strings.Contains(actualStr, "nil = null") {
+		t.Errorf("Expected to have the nil field in message")
+	}
+
+	if !strings.Contains(actualStr, "error = \"field_error\"") {
+		t.Errorf("Expected to have the nil field in message")
+	}
```

Comment:
Nit: do we need a check about how those pieces are put together? "contains" wouldn't catch too many or too few spaces between fields...

Reply:
> do we need a check about how those pieces are put together?

Probably, I'll add in something to check for it a little more than now.